arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -277,7 +277,8 @@ public List<Module> getModulesToTrace() {
         this.shf,
         this.stp,
         this.txnData,
-        this.wcp);
+        this.wcp,
+        this.modexp.data());
```

Comment:
and we put them in alphabetic order

Reply:
I didn't know about that, I will fix it.